software/database/src/main/java/org/apache/brooklyn/entity/database/postgresql/PostgreSqlSshDriver.java

```diff
@@ -320,6 +324,16 @@ private void initializeNewDatabase() {
                 "\"CREATE DATABASE %s OWNER %s\"",
                 StringEscapes.escapeSql(getDatabaseName()),
                 StringEscapes.escapeSql(getUsername()));
+
+        String createRolesAdditionalCommand = "";
+
+        if (!entity.getConfig(PostgreSqlNode.ROLES).isEmpty()) {
```

Comment:
This looks like it will throw an NPE if `ROLES` config value has not been set (because it has no default value, such as an empty immutable map). Am I right?